.github/workflows/2-feladat.yml

```diff
@@ -15,13 +15,13 @@ jobs:
       - name: Gpp Error
         run: g++ -Wall -Werror *.cpp
       - name: Valgrind Error
-        run: valgrind --error-exitcode=1 ./a.out kakarott.json makarott.json
+        run: valgrind --leak-check=full --error-exitcode=1 ./a.out kakarott.json makarott.json
       - name: Cppchecks Error
-        run: cppcheck *.cpp --enable=warning --error-exitcode=1
+        run: cppcheck *.cpp --enable=style --error-exitcode=1
       - name: Error List
-        run: cppcheck *.cpp --enable=warning --output-file=list.txt
+        run: cppcheck *.cpp --enable=performance --output-file=cppcheck.txt
```

Comment:
Az első hívásnál elég lett volna a warning szint, mert arra kell hibával leállítani a workflow-t, és a másodikhoz kellett volna felsorolni a style és performance kategóriákat (bár talán a performance amúgy is bekapcsolja a style-t)